runtime/Cargo.toml

```diff
@@ -20,7 +20,7 @@ log = "0.4.2"
 memmap = "0.6.2"
 rand = "0.6.5"
 rayon = "1.0.0"
-serde = "1.0.88"
+serde = { version = "1.0.88", features = ["rc"] }
```

Comment:
this looks like it might be a really nasty thing to turn on...

Reply:
I will work on serializing / deserializing with out having to turn on this feature.